media/js/firefox/australis/australis-uitour.js

```diff
@@ -177,15 +234,28 @@ if (typeof Mozilla == 'undefined') {
         });
     };
 
-    Mozilla.UITour.showFirefoxAccounts = function() {
-        _sendEvent('showFirefoxAccounts');
+    /**
+     * Request the browser open the Firefox Accounts page.
+     *
+     * @param {Object} extraURLCampaignParams - An object containing additional
```

Comment:
Will do for sure. I think one more iteration of this file is coming down today. Will add docs after next update.